data/categories.txt

```diff
@@ -5027,7 +5028,7 @@ sw:^Ghuba
 natural-spring
 en:^Spring
 ru:^Родник
-ar:^خليجc
+ar:^ربيع|طبيعة
```

Comment:
No! It is this type of spring: https://en.wikipedia.org/wiki/Spring_(hydrology).